packages/models-library/src/models_library/projects.py

```diff
@@ -82,6 +86,20 @@ class Config:
         extra = Extra.forbid
 
 
+class WorkbenchUI(BaseModel):
+    position: Optional[Position] = Field(...)
+
+    class Config:
+        extra = Extra.forbid
+
+
+class Slideshow(BaseModel):
+    position: Optional[int] = Field(..., description="Slide's position", example=["0", "2"])

```

Comment:
these examples are wrong. You need one int, not a list of strings with ints
Is this entry only for positive ints or can be negative? Is there a fixed limit ? Add a constraint e.g. ``PositiveInt``
 SEE [constraints types](https://pydantic-docs.helpmanual.io/usage/types/#constrained-types)
This is the position of the slide in pixels with respect to the upper right screen? please clarify in description

Reply:
@pcrespov : this is not wrong, you can provide more than one example

Reply:
@sanderegg I know you can create [schema customizations](https://pydantic-docs.helpmanual.io/usage/schema/#schema-customization) but 
  a) I notice that it does not play well with swagger UI (it is not generating fakes correctly)
  b) I would rather use ``Config`` example**S** entry
  c) I do not think that example of positive indices are very useful